src/program/snabbvmx/doc/README.configuration.md

```diff
@@ -0,0 +1,169 @@
+# Configuration
+
+## SnabbVMX's configuration file
+
+SnabbVMX has its own configuration file (Example: **snabbvmx-lwaftr-xe0.cfg**).
+Configuration file is actually a Lua file, that is processed by `snabbvmx lwaftr`
+command.
+
+It contains a reference to a Snabb's **lwAFTR** configuration file (which
+contains a reference to a binding-table).
+
+**snabbvmx-lwaftr-xe0.cfg**
+
+```lua
+return {
+   lwaftr = "snabbvmx-lwaftr-xe0.conf",
```

Comment:
I see your point but it's how it normally goes. For instance, the config files in tests/conf follow the same pattern. On the other hand, this pattern has the advantage that it makes it easier to know what's the correspondent Snabb's lwAFTR config file for a SnabbVMX configuration.